tst/standard/main.tst

```diff
@@ -0,0 +1,29 @@
+gap> A := NullMat(3, 3, GF(19));;
```

Comment:
The tests fail and I think this is due to whitespace differences. You can call `Test("tst/standard/main.tst", rec(rewriteToFile := true));` to let GAP generate the "correct" output itself.

I have just recently learned about this nice trick. :)

Reply:
Nice, thanks! I already had problems with that when testing on my computer...